symplyphysics/definitions/angular_velocity_is_angle_derivative.py

```diff
@@ -0,0 +1,40 @@
+from symplyphysics import (
+    symbols, Function, Derivative, Eq, pretty, Quantity, units, pi, SI,
+    validate_input, validate_output, expr_to_quantity
+)
+
+from sympy.physics.units.definitions.dimension_definitions import angle as angle_type
+
+# Description
+## The movement along circle might be easily represented in polar coordinates with the pole in the center of the circle.
+## Object position is described by radius-vector from pole to object. Length of this vector is radius of circle and
+## angle of this vector and it is also known as movement phase.
+
+# Definition: ω(t) = φ(t)/dt, where
+## ω(t) is angular velocity function of time
+## φ(t) is angle function of time
+
+time = symbols('time')
+angular_velocity_function, angle_function = symbols('angular_velocity_function angle_function', cls = Function)
+
+definition = Eq(angular_velocity_function(time), Derivative(angle_function(time), time))
+
+definition_dimension_SI = units.radian / units.second
+
+def print():
+    return pretty(definition, use_unicode=False)
+
+def print_dimension():
+    return pretty(definition_dimension_SI, use_unicode=False)
+
+@validate_input(phase_start_= angle_type, phase_end_= angle_type, moving_time_=units.time)
+@validate_output(1 / units.time)
+def calculate_velocity(phase_start_: Quantity, phase_end_: Quantity, moving_time_: Quantity) -> Quantity:
```

Comment:
We should hint that angle is a linear function. So some better name for the `calculate_velocity` should be considered.